bin/repeat-perf-test.sh

```diff
@@ -37,7 +37,7 @@ if [[ "$#" -lt 1 ]]; then
       Repeatedly run the performance test suite against one or more versions of the codebase.
 
     USAGE
-      [PERF_REPEATS=<N>] $0 ...tree-ish
+      [PERF_REPEATS=<N>] $0 ...tree-ish:adapter
```

Comment:
Which adapters are allowed?

Reply:
https://github.com/pouchdb/pouchdb/blob/master/TESTING.md#adapters-default-depends-on-client

Reply:
> Comma-separated list of preferred adapter backends that PouchDB will use for local
> databases. These are selected automatically based on the execution environment,
> but this variable overrides the default choice and causes additional adapters to
> be loaded if they're not part of the default distribution.                    
>                                                                               
> On Node.js the available local adapters are `leveldb` and `memory`. In the    
> browser they're `idb`, `indexeddb` and `memory`.                              
>                                                                               
> You can also set `ADAPTERS=http` to force all PouchDB databases to be created on
> a remote server, identified by `COUCH_HOST`. This is not necessary for        
> integration tests since they use a mixture of local and remote databases to   
> check compatibility, but it's useful for the `find` and `mapreduce` suites.

Reply:
Thx, seems to be working well.